src/Umbraco.Web/PropertyEditors/MediaPickerConfiguration.cs

```diff
@@ -8,6 +8,9 @@ namespace Umbraco.Web.PropertyEditors
     /// </summary>
     public class MediaPickerConfiguration : IIgnoreUserStartNodesConfig
     {
+        [ConfigurationField("notice", "You can NOT change the property editor", "obsoletemediapickernotice")]
+        public bool Notice { get; set; }
+
```

Comment:
@nul800sebastiaan I think this should be defined as `string`.

Reply:
sure but it doesn't really matter, it's not a REAL prevalue, just a hack to show a warning for now while we think of a betters solution. 😄